docs/ReleaseNotes.txt

```diff
@@ -1,3 +1,113 @@
+=== v1.25.0 === (TBD October 2024)
+
+ISPC release featuring new attribute syntax support, extended template handling
+for short vectors and arrays, added support for new Intel GPUs and CPUs, and
+significant code generation enhancements.  This release is based on patched LLVM
+18.1.8, with the minimum required glibc version for Linux binaries raised to
+2.27.
+
+Language changes:
+
+ - Added support for `__attribute__` syntax in variable and function
+   declarations.  The following attributes are now supported: `noescape`,
+  `address_space(N)`, `external_only`, and `unmangled`.
+ - Extended template support for short vectors and array declarations.  You can
+   now use type and non-type parameters to specify the type and dimensions of
+   these types.
+ - Added error messages for variable declarations with incompatible type
+   specifiers.
+ - Supported typedef with same name as struct tag.
+
+Code generation:
+
+ - Switched to the NewGVN pass for global value numbering for CPU targets. This
```

Comment:
We need to remove `NewGVN` entry because of #3056.

Reply:
Updated